LineBackground animation 컴포넌트를 정의하고 적용한다

dididy opened the pull request:
* mobile breakpoint를 위한 sinppet을 정의한다 6572d81
* LineBackground animation 컴포넌트를 정의하고 적용한다 6fe0f82
  * Landing animation에 필요한 stickySlice 를 정의한다 fead77f
  * 애니메이션 효과를 위한 scenejs를 모듈을 설치한다 248691d
  * useWindowResize, useWindowScroll 등의 커스텀 훅을 정의한다 05ccd21


## Reference
https://tobbelindstrom.com/blog/how-to-create-a-breakpoint-mixin-with-styled-components/
https://github.com/fedgkr/feconf2020/tree/master/src/views/svg-components/LineBackground

dididy committed:
feat: mobile breakpoint를 위한 sinppet을 정의한다

* style 폴더에 위치시킴
https://tobbelindstrom.com/blog/how-to-create-a-breakpoint-mixin-with-styled-components/

dididy committed:
chore(env): 애니메이션 효과를 위한 scenejs를 모듈을 설치한다

https://codepen.io/collection/DLWxrd/

dididy committed:
feat: useWindowResize, useWindowScroll 등의 커스텀 훅을 정의한다

https://github.com/fedgkr/feconf2020/blob/master/src/utils/hooks/use-window.ts

dididy committed:
feat: LineBackground animation 컴포넌트를 정의하고 적용한다

https://github.com/fedgkr/feconf2020/tree/master/src/views/svg-components/LineBackground

Dongkyun-Jang commented on hooks/useWindow.ts:
```diff
@@ -0,0 +1,179 @@
+import { useRef, useEffect, MutableRefObject, useState } from 'react';
```
이런 식으로 커스텀 훅을 만들어서 간편하게 사용할 수도 있군요.....ㄷㄷ
역시...갓

dididy replied:
feconf 코드 배울게 엄청 많은거같아요!

miro-ring reviewed (approved):
또 공부해야할 좋은 코드들이 늘었군요!!! ㅋㅋㅋ
감사합니다 ㅎㅎㅎ

dididy committed:
feat: Landing animation에 필요한 stickySlice 를 정의한다

https://github.com/fedgkr/feconf2020/blob/master/src/store/slices/stickySlice.ts

dididy committed:
Merge branch 'main' of https://github.com/ssafy-null/fe into feature/landing
